packages/share_plus/share_plus/android/src/main/kotlin/dev/fluttercommunity/plus/share/ShareSuccessManager.kt

```diff
@@ -36,13 +33,17 @@ internal class ShareSuccessManager(private val context: Context): BroadcastRecei
      * the componentname of the chosen option or an empty string on dismissal.
      */
     fun setCallback(callback: MethodChannel.Result): Boolean {
-        if (calledBack.compareAndSet(true, false)) {
+        return if (calledBack.compareAndSet(true, false)) {
```

Comment:
While this does do the job, don't you agree that this makes the code less readable? We don't really get anything from this change, and it's not consistent (see lines 72-77).

Reply:
No, I don't think so. It is a normal Kotlin syntax, which is used for a long time in Android development. If you would use Android Studio you would even see a lint suggestion to change this way. 
Lines you suggested to consider are a different case, which can't be turned into construction like here.

Reply:
Well, you could just do this on lines 72-77:
```kotlin
return if (requestCode == ACTIVITY_CODE) {
    returnResult("")
    true
} else {
    false
}
```

Reply:
> Well, you could just do this on lines 72-77 or am I wrong?

You are right, I overlooked that case.

> guess I just don't like how it looks 🤷🏻‍♂️

Just a case of tastes, I guess, because for me as for the person, who was working with Kotlin for a long time it seems fine. Will update code for consistency, though.